pymc3/tuning/starting.py

```diff
@@ -97,6 +97,14 @@ def find_MAP(
             "Valid keys are: {}, but {} was supplied".format(valid_keys, extra_keys)
         )
 
+    initial_eval = model.check_test_point(test_point=start)
+
+    if not np.all(np.isfinite(initial_eval)):
```

Comment:
Feel free to ignore if you don't agree it is more readable.
```suggestion
    if np.any(~np.isfinite(initial_eval)):
```